### PR TITLE
build: enable size plugin from angular robot

### PR DESCRIPTION
**Pull request by DevVersion**
We only measure the CDK and Material for now. We can revisit checking other packages as well in the future. First, we should figure out whether the size plugin works for us.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -200,6 +200,23 @@ jobs:
...
             - "**/*"
 
+      # Since there is no UMD bundle that includes everything from the CDK, we need to move
+      # all bundles into a directory. This allows us to store all CDK UMD bundles as job
+      # artifacts that can be picked up by the Angular Github bot.
+      - run:
+          name: Prepare CDK artifacts for publish.
+          command: |
+            mkdir -p  /tmp/cdk-umd-artifacts
+            cp dist/releases/cdk/bundles/**/*.umd.js /tmp/cdk-umd-artifacts
```
Comment (josephperrott) [inline .circleci/config.yml]: Looks like `dist/releases/cdk/bundles/**/*.umd.js` doesn't exist during the circle run.

Comment (DevVersion) [reply .circleci/config.yml]: Yeah, I already force pushed. Thanks

File: .circleci/config.yml [hunk 2]
```diff
@@ -200,6 +200,25 @@ jobs:
...
+      # Since there is no UMD bundle that includes everything from the CDK, we need to move
+      # all bundles into a directory. This allows us to store all CDK UMD bundles as job
+      # artifacts that can be picked up by the Angular Github bot.
+      - run:
+          name: Prepare CDK artifacts for publish.
+          command: |
+            mkdir -p  /tmp/cdk-umd-artifacts
+            cp dist/releases/cdk/bundles/*.umd.js /tmp/cdk-umd-artifacts
+
+      # Publish artifacts in favor of the CircleCI size calculations.
```
Comment (jelbourn) [inline .circleci/config.yml]: The phrase "in favor of" is usually used when comparing two things. I think you mean "for use by".

Comment (DevVersion) [reply .circleci/config.yml]: Makes sense. Updated

**Pull request merged**